google/ggrc-core
# GGRC-4732 Fix old migrations from ggrc_risks

# Issue description

During the implementation of the migration for the document epic I've figured out that ggrc_risks migration chain contains schema migration for the documents table. And this migration conflicting with my, because of the same table. After short investigation I've found that same table scheme is presents in ggrc migration chain that is odd.


# Steps to test the changes
Check that db_reset passes w/o errors. Check that documents table is the same as it was before this PR.

# Solution description
I've removed unnecessary migration code from ggrc_risks and refactored util function.

# Sanity checklist


# Migration checklist


# PR Review checklist
